easybuild/tools/config.py

```diff
@@ -508,11 +512,11 @@ def package_path():
     """
     return ConfigurationVariables()['packagepath']
 
-def singularity_path():
+def container_path():
     """
-    Return the path for singularity definition and image directory are copied to
+    Return the path for container recipes & images
     """
-    return ConfigurationVariables()['singularitypath']
+    return ConfigurationVariables()['containerpath']
```

Comment:
@boegel 

it appears that containerpath is not modified when using configuration file

```
[siddis14@amrndhl1157 easybuild-framework]$ cat ~/.local/easybuild/config.cfg
[config]
containerpath = /tmp
[siddis14@amrndhl1157 easybuild-framework]$ eb --show-full-config | grep containerpath
containerpath                             (D) = /home/siddis14/.local/easybuild/containers

```

It is working via command line and environment variable

```
[siddis14@amrndhl1157 easybuild-framework]$ export EASYBUILD_CONTAINERPATH=/tmp
[siddis14@amrndhl1157 easybuild-framework]$ eb --show-full-config | grep containerpath
containerpath                             (E) = /tmp
```

This feature was working before. Can you please look into this.

Reply:
@shahzebsiddiqui The default configuration file is in `$HOME/.config/easybuild/config.cfg`, not `$HOME/.local/easybuild/config.cfg`, see http://easybuild.readthedocs.io/en/latest/Configuration.html#default-configuration-files and the output of `eb --show-default-configfiles`

works fine if you use the right location:

```
$ cat ~/.config/easybuild/config.cfg
[config]
containerpath = /tmp

$ eb --show-config | grep containerpath
containerpath         (F) = /tmp
```

Reply:
You are right, thanks for the clarification. I got the paths mixed up. This is working.